config/crd/bases/config.k8ssandra.io_clientconfigs.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.2
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
What's the origin of this? The changes get reverted when I build locally.

Reply:
It's because I'm using a version of controller-gen that's not the latest:

```
% controller-gen --version
Version: v0.8.0
```

no big deal.